modules/s3/s3codecs/pdf.py

```diff
@@ -90,6 +91,17 @@
 PDF_WIDTH = 0
 PDF_HEIGHT = 1
 
+if current.response.s3.language == "tr":
```

Comment:
@flavour: In some languages (eg. - English), which use the normal font (maybe ~Helevitca..) there is a bold version for that font, which is used at some places, how would I deal with that case ? *Multiple fonts for a single language*
Also Is there any bold version for the unifont too, It might look good for the headers ? 
I couldn't find any on the internet.